### ISSUE TITLE
Audit the docs directory for accessibility 

### ISSUE THREAD

**Issue by christinach** [pulibrary/bibdata#2175]
Use the [accessibility audit procedure](https://github.com/pulibrary/dacs_handbook/blob/main/Accessibility/audit_procedure.md) to audit all pages in the /docs directory of this repository.  Address any accessibility issues.  Create github issues for any issues that would take longer than 1 day to address.

**Comment by lizgarciao** [comment]
Checked the following documents with axe DevTools and Voice over and included any issues found on [report template](https://docs.google.com/spreadsheets/d/1YSJ-MBwQzI20JT02xRgvQEa2cZl68byuTS4J_eL25Zo/edit#gid=0).

1. action_notes.md
2. add-missing-dump-to-next-event.md
3. alma.md
4. alma_publishing_jobs_schedule.md
5. api_endpoints.md
6. augment_the_subject.md
7. bad_dump.md
8. browse-list.md

**Comment by christinach** [comment]
@lizgarciao I'll turn this issue to an epic. Feel free to create github tickets for the issues you added in the audit report. Then add them in this epic. Thanks! Great work 😃
